SharpBgfx/Texture.cs

```diff
@@ -105,18 +113,20 @@ public unsafe sealed class Texture : IDisposable, IEquatable<Texture> {
         /// </summary>
         /// <param name="ratio">The amount to scale when the backbuffer resizes.</param>
         /// <param name="mipCount">The number of mip levels.</param>
+        /// <param name="hasMips">Indicates that texture contains full mip-map chain.</param>
+        /// <param name="numLayers">Number of layers in texture array. Must be 1 if Texture2DArray caps flag not set.</param>
         /// <param name="format">The format of the texture data.</param>
         /// <param name="flags">Flags that control texture behavior.</param>
         /// <returns>
         /// The newly created texture handle.
         /// </returns>
-        public static Texture Create2D (BackbufferRatio ratio, int mipCount, TextureFormat format, TextureFlags flags = TextureFlags.None) {
+        public static Texture Create2D (BackbufferRatio ratio, bool hasMips, int numLayers, TextureFormat format, TextureFlags flags = TextureFlags.None) {
             var info = new TextureInfo {
                 Format = format,
-                MipCount = (byte)mipCount
+                Layers = (ushort)numLayers
```

Comment:
Probably we need to manually calculate MipCount, otherwise the property will always report 0. I'll look around at what the native code does to fill in that member.